package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "docz dev",
     "build": "docz build",
     "serve": "docz build && docz serve",
-    "deploy": "PUBLIC_URL=https://ethereum.github.io/js-organization/ docz build && gh-pages -d './.docz/dist'"
+    "deploy": "DOCZ_PATH=/js-organization/ docz build && gh-pages -d './.docz/dist'"
```

Comment:
`DOCZ_BASE`, no? You'll only be able to see a deploy in earnest if there's a change to the content or metadata of the docs.

https://www.docz.site/docs/project-configuration#base

Reply:
correct, forgot to push